src/config/graphical-session/gnome.md

```diff
@@ -2,30 +2,52 @@
 
 ## Pre-installation
 
+*[TODO: Maybe point to Xorg and Wayland setup?]*
+
 Install the `dbus` package, ensure the `dbus` service is enabled, and reboot for
 the changes to take effect.
 
 ## Installation
 
-Install the `gnome` package for a GNOME environment which includes GNOME
-applications.
+Install the `gnome` package for a GNOME environment which includes the base
+GNOME desktop and a subset of GNOME applications. Additional applications are
+available via the `gnome-apps` package.
 
-A minimal GNOME environment can be created by installing the `gnome-session`,
-`gdm` and `adwaita-icon-theme` packages. (Note, however, that not all GNOME
-features may be present or functional.) `gdm` defaults to providing a Wayland
-session, via the `mutter` window manager. For an Xorg session, install the
-`xorg` package, then select 'GNOME on Xorg' at the GDM login screen, or start
-`gnome-session` via `~/.xinitrc`. GNOME applications can be installed via the
-`gnome-apps` package.
+A minimal GNOME environment is available via the `gdm` package. (Note, however,
+that not all GNOME features may be present or functional.)
 
 If you require [ZeroConf](http://www.zeroconf.org/) support, install the `avahi`
 package and enable the `avahi-daemon` service.
 
-## Display Manager
+## Starting GNOME
+
+### From Display Manager
```

Comment:
What do you think about using "From the Display Manager" and "From the Console"? I'm not sure they are better, but they might be.

Reply:
Sounds better with `the` imo